@xen-orchestra/immutable-backups/config.sample.toml

```diff
@@ -0,0 +1,14 @@
+
+liftEvery =  1 # in hours, set to 0 if you want to handle it by another way, like cron
```

Comment:
You can use the lib `ms` for duration, example: https://github.com/vatesfr/xen-orchestra/blob/d6abdb246bba9dae10a6ee71166bca2d574790a1/packages/xo-server/config.toml#L8